analyzers/tests/SonarAnalyzer.UnitTest/TestCases/TooManyParameters_CustomValues.CSharp12.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+public class MyWrongClass(int p1, int p2, int p3, int p4) { } // FN {{Constructor has 4 parameters, which is greater than the 3 authorized.}}
```

Comment:
Reproducer for #8143

Reply:
Since there is already a fix ongoing, Adding a link to an issue and a namespace with the name of the issue reproduced would just make unnecessary noise. So it's good to just add the test scenario.